doc/uninstall.rst

```diff
@@ -0,0 +1,54 @@
+===============================================
+Uninstalling the Desktop Synchronization Client
+===============================================
+
+Uninstalling the synchronization client isn't always a completely straight-forward process as, sometimes, artifacts can be left behind once the official uninstaller processes completes.
+
+This section of the documentation is intended to guide you through ensuring that, once the official uninstaller processes completes, that you are able to ensure that you can both find an remove any leftover artifacts, ensuring that your system is clean.
+
+Please refer to the section for your operating system to find detailed instructions.
```

Comment:
Since there are no sections for Linux or Mac (yet?), it would be good to mention that either no special extra uninstall is needed, or that those sections are not yet written.